examples/cosmos/bank/bankTest.qnt

```diff
@@ -96,12 +98,13 @@ module bankTest {
       }
     }
   }
+
   run sendTest = {
     init
       // the King has bananas and he can send it
       .then(send("king",
                  "donkeykong",
-                 Set("banana").mapBy(d => 2_000)))
+                 Map("banana" -> 2_000)))
```

Comment:
Thanks! This indeed looks much simpler :)

Reply:
I just realized that I need to update the markdown, and then re-generate the file. I'm doing that right now!